src/app/code/community/Shopgate/Cloudapi/etc/api2.xml

```diff
@@ -42,6 +42,10 @@
                                 <title>Addresses</title>
                                 <sort_order>10</sort_order>
                             </shopgate_cloudapi_v2_customers_addresses>
+                            <shopgate_cloudapi_v2_customers_email translate="title" module="Shopgate_Cloudapi">
```

Comment:
I think this needs to move in (with the password) together under user info section:
<img width="332" alt="screen shot 2018-09-19 at 12 23 29" src="https://user-images.githubusercontent.com/844601/45749219-3edaf800-bc0b-11e8-8052-dc26b3bccf13.png">